VmAgent.Core/Interfaces/DockerContainerEngine.cs

```diff
@@ -556,31 +568,30 @@
```

Comment:
this was here before, but in any case this is not an accurate comment. We are not deleting the image (docker rmi command) but rather the instance of the image (the actual container), so I'd remove the "image" from the log.